### PR TITLE
Add option to override ring position via command line arguments or the config file

### PR DESCRIPTION
**Pull request by matt-della**
I got everything sorted out on my end, so here's my first PR! 

Closes: https://github.com/swaywm/swaylock/issues/106

### CODE REVIEW

File: render.c [hunk 1]
```diff
@@ -80,10 +80,27 @@ void render_frame(struct swaylock_surface *surface) {
...
-		(state->args.radius + state->args.thickness);
+	int subsurf_xpos;
+	int subsurf_ypos;
+
+	// Center the indicator unless overridden by the user
+	if (state->args.override_indicator_x_position) {
+		subsurf_xpos = state->args.indicator_x_position -
+			buffer_width / (2 * surface->scale) + 2 / surface->scale;
+	}
+	else {
```
Comment (ddevault) [inline render.c]: style: this goes on the previous line

**Comment by ddevault** [comment]
LGTM

File: render.c (hunk 1 above)
Comment (matt-della) [reply render.c]: Fixed. See commit f0bcc2f.

**Comment by ddevault** [comment]
Can you squash your commits?

**Comment by matt-della** [comment]
I certainly hope I did that right. Apologies for my lack of experience here, I've never done anything with Git beyond the standard "git init, git add, git commit" workflow, so I'm somewhat fumbling my way through this process.

**Comment by ddevault** [comment]
Not quite right, but that's okay, you can still fix it.

https://git-rebase.io would be helpful here.

**Comment by matt-della** [comment]
Okay, I may have fumbled my way through this. Please let me know if I was successful in my efforts. 
My understanding is that I created a branch, performed an interactive rebase operation, picked one commit, and squashed the others, thus squashing my changes into a single, atomic commit.
Hopefully it did what I was hoping it would do.

**Comment by ddevault** [comment]
Bleh, not quite.

    git remote add upstream https://github.com/swaywm/swaylock
    git fetch upstream
    git rebase upstream/master
    git rebase -i upstream/master

Then change all but the first line to "squash".

**Comment by matt-della** [comment]
Okay, so I may have used a sledgehammer approach here, but I think it worked. Fortunately, I had a pristine copy of my repo from just before I committed anything, so I followed your steps using that and then forced the push operation to get everything back in sync. I have high hopes for this one. 

I'm quickly figuring out that I may need to force myself to learn this tool a little better, so thank you so much for being patient with me while I figure things out.

**Comment by ddevault** [comment]
Thanks!

**Pull request merged**